website/markdown/docs/plugins/using-plugins.mdx

```diff
@@ -0,0 +1,88 @@
+---
+name: Using plugins
+excerpt: Learn how to extend Tuist with plugins.
+---
+
+# Using Plugins
+
+## Plugins
+
+Plugins are meant to be separate and isolated directories, each containing code used to extend existing Tuist functionality. 
+You may use plugins that exist at a path local to your machine or at a git source URL to enable sharing and version control across multiple projects.
+
+## Adding plugins
+
+In order to tell Tuist you'd like to use plugins in your project, you can start by defining the plugins in the [`Config.swift`](/docs/usage/config/) manifest.
+
+```swift
+import ProjectDescription
+
+let config = Config(
+    plugins: [
+        .local(path: "/Plugins/MyPlugin"),
+        .git(url: "https://url/to/plugin.git", tag: "1.0.0"),
+        .git(url: "https://url/to/plugin.git", sha: "e34c5ba")
+    ]
+)
+```
+
+By defining plugins in the [`Config.swift`](/docs/usage/config/) manifest Tuist is able to collect, cache and use your plugins to add or extend functionality.
+
+### Local plugin
+
+Local plugins are stored relative to the machine generating the project. You can tell Tuist to find them at a given [`Path`](/docs/usage/project-description/). 
+
+```swift
+PluginLocation.local(path: "/Plugins/MyPlugin")
+PluginLocation.local(path: .relativeToCurrentFile("MyOtherPlugin/"))
+PluginLocation.local(path: .relativeToManifest("../DevelopmentPlugin"))
+```
+
+### Remote plugin
+
+Remote plugins are stored on a remote server and can be downloaded by Tuist and shared across multiple projects.
+
+#### git
```

Comment:
I have the feeling that the git part assumes that the Plugin.swift file is in the root of the repository (I haven't checked the implementation to confirm this assumption).

If this is the case, maybe it is worth specifying it. It may be obvious, but one can think of having a repo of multiple plugins, each on a sub folder of the repo (and I guess this use case is not supported). I'd do it also because a plugin is defined as "a folder with a plugin.swift file", without mentioning any additional requirement

Reply:
You're correct it must be at the root in the current implementation, adding this!